squad/frontend/static/main.css

```diff
@@ -212,3 +212,25 @@ table.test-results-details {
 .search-dropdown-display:hover {
   background: #eee;
 }
+
+.fullscreen-button {
+  border-radius: 3px;
+  border: none;
+  font-size: 0.9rem;
+  background: #337ab7;
+  border-bottom: 1px solid #498b50;
+  color: white;
+  -webkit-font-smoothing: antialiased;
+  font-weight: bold;
+  text-align: center;
+}
+
+.fullscreen-button:hover, .fullscreen-button:focus {
+  opacity: 0.75;
+  cursor: pointer;
+}
+
+.fullscreen-button:active {
+  opacity: 1;
+  box-shadow: 0 -3px 10px rgba(0, 0, 0, 0.1) inset;
+}
```

Comment:
do we really need all this CSS? I would rather have a regular bootstrap button

Reply:
also, font awesome has a fullscreen icon, maybe we could use that? 

My suggestion is to remove the in-chart title, turn that into a `<h2>`, and put the full screen button close to that header. and move the "Toggle full screen" to the title attribute of the button